shell/src/main/java/org/apache/accumulo/shell/commands/ActiveCompactionHelper.java

```diff
@@ -94,31 +99,41 @@ private static String formatActiveCompactionLine(ActiveCompaction ac) {
     try {
       var dur = new DurationFormat(ac.getAge(), "");
       return String.format(
-          "%21s | %9s | %5s | %6s | %5s | %5s | %15s | %-40s | %5s | %35s | %9s | %s", host, dur,
-          ac.getType(), ac.getReason(), shortenCount(ac.getEntriesRead()),
-          shortenCount(ac.getEntriesWritten()), ac.getTable(), ac.getTablet(),
-          ac.getInputFiles().size(), output, iterList, iterOpts);
+          "%21s | %21s | %9s | %5s | %6s | %5s | %5s | %15s | %-40s | %5s | %35s | %9s | %s",
+          ac.getHost().getResourceGroup(), host, dur, ac.getType(), ac.getReason(),
+          shortenCount(ac.getEntriesRead()), shortenCount(ac.getEntriesWritten()), ac.getTable(),
+          ac.getTablet(), ac.getInputFiles().size(), output, iterList, iterOpts);
     } catch (TableNotFoundException e) {
       return "ERROR " + e.getMessage();
     }
   }
 
   public static Stream<String> appendHeader(Stream<String> stream) {
     Stream<String> header = Stream.of(String.format(
-        " %-21s| %-9s | %-5s | %-6s | %-5s | %-5s | %-15s | %-40s | %-5s | %-35s | %-9s | %s",
-        "SERVER", "AGE", "TYPE", "REASON", "READ", "WROTE", "TABLE", "TABLET", "INPUT", "OUTPUT",
-        "ITERATORS", "ITERATOR OPTIONS"));
+        " %-21s| %-21s| %-9s | %-5s | %-6s | %-5s | %-5s | %-15s | %-40s | %-5s | %-35s | %-9s | %s",
+        "GROUP", "SERVER", "AGE", "TYPE", "REASON", "READ", "WROTE", "TABLE", "TABLET", "INPUT",
+        "OUTPUT", "ITERATORS", "ITERATOR OPTIONS"));
     return Stream.concat(header, stream);
   }
 
   public static Stream<String> activeCompactionsForServer(String tserver,
       InstanceOperations instanceOps) {
-    try {
-      return instanceOps.getActiveCompactions(tserver).stream().sorted(COMPACTION_AGE_DESCENDING)
-          .map(ActiveCompactionHelper::formatActiveCompactionLine);
-    } catch (Exception e) {
-      Shell.log.debug("Failed to list active compactions for server {}", tserver, e);
-      return Stream.of(tserver + " ERROR " + e.getMessage());
+    final HostAndPort hp = HostAndPort.fromString(tserver);
+    ServerId server =
```

Comment:
Not sure if it possible in 2.1. If it is it would be nice to update the 2.1 command to list compactions for a single compactor.

Reply:
The 2.1 ListCompactionsCommand has the ability for the user to specify a specific TServer or Compactor.